Algebraic Topology/algebraic-topology.tex

```diff
@@ -454,9 +454,94 @@ \section{Functoriality of the fundamental group}
 \end{proof}
 
 
+\mydate{d7}{8}{2}{2017}
 
+\section{CW complexes and the fundamental group}
 
+Consider the polygon
 
+\begin{figure}
+  % \includegraphics{}
+  \label{fig:star}
+\end{figure}
+
+$X_0$ = picture
+
+$X_1$ = picture ((it seems I am inconsistent with whether these are capital x's or lowercase.  Should they all be capital?  I went back and made them all capital.))
```

Comment:
Yes, I think the convention is that when it is a topological space, it is denoted by a capital letter.